install.sh

```diff
@@ -200,6 +204,38 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+# Please sync with 'test-ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
+test-ruby () {
```

Comment:
```suggestion
test_ruby () {
```
I believe our convention is to use `_` rather than `-` in function names.

Reply:
I copied this function directly from `Library/Homebrew/utils/ruby.sh`. It's better to keep these two functions identical so that `install.sh` does exactly the same check as `ruby.sh`